### PR TITLE
Update metadata latex_labels.py

### PR DESCRIPTION
**Pull request by KesharwaniArpita**
It's better to use the terminology of "graph" or "Plot" or grid in mathematical and Visualization context. Increased the size of the graph to make it look cleaner.

### CODE REVIEW

File: examples/models/latex_labels.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-''' A plot showing LaTeX ``Label`` objects in many different locations,
-inside and outside the figure.
+''' A plot showing LaTeX ``Label`` objects in many different point,
```
Comment (mosc9575) [inline examples/models/latex_labels.py]:
```suggestion
''' A plot showing LaTeX ``Label`` objects in many different points,
```

Maybe "positions" could be another option.

File: examples/models/latex_labels.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from bokeh.models import Label
 from bokeh.plotting import figure, show
 
-p = figure(width=600, height=600, x_range=(0, 10), y_range=(0, 10))
+p = figure(width=600, height=600, x_range=(0, 20), y_range=(0, 20))
```
Comment (mosc9575) [inline examples/models/latex_labels.py]: Here is the plot how it looked before and after your changes.

|`figure(width=600, height=600, x_range=(0, 10), y_range=(0, 10))`|`figure(width=600, height=600, x_range=(0, 20), y_range=(0, 20))`|
|--|--|
|grafik|grafik|

I think this is a mistake and should be reverted.

Comment (KesharwaniArpita) [reply examples/models/latex_labels.py]: Yeah! I guess I overlooked that

**Comment by ianthomas23** [comment]
We tend to use the words `figure` and `plot` as those are the API function names. `graph` means something quite specific in a computer science context (https://docs.bokeh.org/en/latest/docs/user_guide/topics/graph.html#ug-topics-graph) so I don't think that is an improvement.

**Comment by bryevdv** [comment]
I have to agree, it is preferable to keep terminology consistent. Also the primary purpose of this code is to be visual comparison, so it also not advised to alter the size. 

Thanks for the PR @KesharwaniArpita but we will decline this one.

**Comment by KesharwaniArpita** [comment]
Okay, got it
Thanks!

**Pull request closed without merging**